asterius/rts/rts.gc.mjs

```diff
@@ -32,22 +35,72 @@ export class GC {
     for (const p of export_stableptrs) this.stablePtrManager.newStablePtr(p);
     this.symbolTable = symbol_table;
     this.reentrancyGuard = reentrancy_guard;
+    /**
+     * 'Yolo' mode disables garbage collection altogether
+     * (see {@link GC#performGC})
+     * @name GC#yolo
+     */
     this.yolo = yolo;
-    this.gcThreshold = gc_threshold;
+    /**
+     * Garbage collection will not be performed when the
+     * current number of MBlocks is less than {@link GC#gcThreshold}
+     * (see {link GC#performGC}).
+     * @name GC#gcThreshold
+     * @default 64
+     */
+    this.gcThreshold = gcThreshold;
+    /**
+     * Map used during evacation, in order to store
```

Comment:
Worth mentioning that the from/to pointers are both `unDynTag`ed (removed lowered 3 bits used by GHC's dynamic pointer tagging).